src/css/Contact.css

```diff
@@ -0,0 +1,27 @@
+.reach{
+padding-top: 1%;
+padding-left: 27%;
+width: 35%;
+height: 23%;
+text-align: left;
+font-family: Arial;
+font-size:  10vh;
```

Comment:
Change this font size to 13.5vh so that it resembles the references font size. Also we've changed the font to Lato.